docs/site/index/cldr-spec/picking-the-right-language-code.md

```diff
@@ -1,92 +1,92 @@
+
+Often it is not clear which language identifier to use. For example, what most people call Punjabi in Pakistan actually has the code '[lah](http://unicode.org/cldr/utility/languageid.jsp?a=lah)', and formal name "Lahnda". There are many other cases where the same name is used for different languages, or where the name that people search for is not listed in the IANA registry. Moreover, a language identifier uses not only the 'base' language code, like '[en](http://unicode.org/cldr/utility/languageid.jsp?a=en)' for English or '[ku](http://unicode.org/cldr/utility/languageid.jsp?a=ku)' for Kurdish, but also certain modifiers such as [en\-CA](http://unicode.org/cldr/utility/languageid.jsp?a=en-CA) for *Canadian English*, or [ku\-Latn](http://ku-Latn) for *Kurdish written in Latin script*. Each of these modifiers are called *subtags* (or sometimes *codes*), and are separated by "\-" or "\_". The language identifier itself is also called a *language tag*, and sometimes a *language code*.
+
+Here is an example of the steps to take to find the right language identifier to use. Let's say you want to find the identifier for a language called "Ganda" which you know is spoken in Uganda. You'll first pick the base language subtag as described below, then add any necessary script/territory subtags, and then verify. If you can't find the name after following these steps or have other questions, ask on the [Unicode CLDR Mailing List](http://www.unicode.org/consortium/distlist.html#cldr_list).
+
+If you are looking at a prospective language code, like "swh", the process is similar; follow the steps below, starting with the verification.
+
+## Choosing the Base Language Code
+
+1. Go to [iso639\-3](https://iso639-3.sil.org/code_tables/639/data) to find the language. In the "Language Name(s) containing" field enter the language name (**Ganda** in this example) and select "Apply".
+2. There may be multiple entries for the item you want, so you'll need to examine them carefully.
+3. You'll find an entry like:
+
+&emsp;lug&emsp; lug&emsp; **lg**&emsp; Ganda&emsp; Individual&emsp; Living
+
+While you may think that you are done, you have to verify that the three\-letter code is correct.
+
+1. Click on the item in the first column ([lug](https://iso639-3.sil.org/code/lug) in this case) and you'll find links to Ethnologue, Glottolog and Wikipedia.
+2. Click on the [Ethnologue](http://www.ethnologue.com/language/lug) link and you get to the corresponding Ethnologue entry.
+3. Verify that is indeed the language:
+	1. Look at the information on the Ethnologue page
+	2. Check Glottolog, Wikipedia and other web sources
+4. ***AND IMPORTANTLY: Review "[Caution](/index/cldr-spec/picking-the-right-language-code#caution)" section below***

```

Comment:
I think you can just use
`[Caution](#caution)`
when linking to an anchor on the same page, but I'm not sure...